libzmix/bulletproofs_amcl/src/r1cs/gadgets/vsmt_4.rs

```diff
@@ -0,0 +1,613 @@
+use super::helper_constraints::constrain_lc_with_scalar;
```

Comment:
Is this done? If so, this uses way more bits than I thought it would. I believed it to be 1 bit/leaf and it looks like 2 bits. Am I wrong?

Reply:
The number 4 is for the branching factor of the tree as its for a 4-ary tree.